src/Form/atoms/DropArea.tsx

```diff
@@ -57,6 +57,15 @@ const onDrop = useCallback((e ) => {
   setInDropZone(false);
 },[dropCallback]);
 
+useEffect(() => {
+  window.addEventListener("dragover",function(e){
+    e.preventDefault();
+  },false);
+  window.addEventListener("drop",function(e){
+    e.preventDefault();
+  },false);
+},[]);
```

Comment:
Just adding listener will keep this added in the window, that's why I suggested to remove it on component unmount. Please check this suggestion.
Also by default `useCapture` the 3rd option of addEventListener is false so no need to specify this.

```suggestion

  const handleWindowDragAndDrop = useCallback((e) => {
    e.preventDefault();
  }, []);

  useEffect(() => {
    window.addEventListener("dragover", handleWindowDragAndDrop);
    window.addEventListener("drop", handleWindowDragAndDrop);
    return () => {
      window.removeEventListener('dragover', handleWindowDragAndDrop);
      window.removeEventListener('drop', handleWindowDragAndDrop);
    };
  }, [handleDragAndDrop]);
```

Reply:
Done